web/src/routes/PoisPage.tsx

```diff
@@ -154,32 +171,52 @@ const PoisPage = ({ cityCode, languageCode, cityModel, pathname, languages }: Ci
       </LocationLayout>
     )
   }
+
   const sortedPois = featureLocations.sort((poi1: PoiFeature, poi2: PoiFeature) =>
     poi1.properties.title.localeCompare(poi2.properties.title)
   )
   const renderPoiListItem = (poi: PoiFeature) => <PoiListItem key={poi.properties.path} properties={poi.properties} />
   const pageTitle = `${t('pageTitle')} - ${cityModel.name}`
 
+  const mapView = cityModel.boundingBox && (
+    <MapView
+      selectFeature={selectFeature}
+      changeSnapPoint={changeSnapPoint}
+      featureCollection={embedInCollection(sortedPois)}
+      bboxViewport={moveViewToBBox(cityModel.boundingBox, defaultViewportConfig)}
+      currentFeature={currentFeature}
+    />
+  )
+
+  const poiList = <List noItemsMessage={t('noPois')} items={sortedPois} renderItem={renderPoiListItem} borderless />
+  // To get the map fullheight its reduced by header, footer, navMenu
```

Comment:
Personal opinion: I think this comment does not really add any necessary information

Reply:
check last commit